### DISCUSSION TITLE
如何使用简悦打开文本（本地）文件

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1689] [使用技巧]
## 重要提示

> ### 此文档同步更新至 [文档中心](https://simpread.pro/docs/#/打开文本文件)。

***

## 问题描述

RT

## 解决方案

1. 打开 `chrome://extensions/`

2. 找到简悦，进入 **详细信息**

3. 勾选 **允许访问文件网址** 

![image](https://user-images.githubusercontent.com/81074/104154621-4e0b9280-5420-11eb-97f6-011be3b617ba.png)

## 引申

> 将本地的任意快照生成阅读模式，同时也加入到简悦的稍后读系统，请使用 [稍后读 · 读取外部文件](https://github.com/Kenshin/simpread/discussions/4625)。

**Comment by menmeng-wong** [comment]
请问edge浏览器怎么设置呀？
